torch/_custom_op/impl.py

```diff
@@ -766,7 +766,7 @@ def create_unbacked_symint(self, *, min=2, max=None) -> torch.SymInt:
             )
 
         result = self._shape_env.create_unbacked_symint()
-        torch.fx.experimental.symbolic_shapes.constrain_range(result, min=2, max=max)
+        torch.fx.experimental.symbolic_shapes.constrain_range(result, min=2, max=max if max else sys.maxsize)
```

Comment:
Is this change required?

I have a good reason to not want to pass sys.maxsize as the upper bound: it will result in extra arbitrary precision integer math in many situations. For example, suppose I have x bounded (2, inf), if I multiple x by two its modified bound is (4, inf) (easy), but if I had maxsize, the modified bound is (4, maxsize * 2) (which is only representable in arbitrary precision.)

Reply:
Ohh this is not needed anymore. It was left from previous iteration.